### ISSUE TITLE
regression: podman run --privileged: runs as container_t

### ISSUE THREAD

**Issue by edsantiago** [containers/libpod#1575]
```
# podman run --rm --privileged --userns=host alpine cat /proc/self/attr/current
system_u:system_r:container_t:s0:c125,c328       <<< expected spc_t
```
podman-0.9.3.1-1.git1cd906d.fc28 and .fc29.

Note that running with `--security-opt label=disable` still works as expected (`spc_t`).

**Comment by mheon** [comment]
Should be fixed by #1576

**Comment by edsantiago** [comment]
Confirmed fixed in podman-0.10.1-1.gite4a1553.fc29
